gulpfile.ts

```diff
@@ -55,6 +56,10 @@ gulp.task('run_server', () => {
   });
 });
 
+gulp.task('clean', (callback) => {
+    return del('out/*', {force: true});
```

Comment:
I think it's better to use 'out' instead of 'out/*'.
We want to remove the out directory as well as out/* files(and sub-directories).

Also, could you write a test for this patch?
You can write the test in bootstrap/test/clean.test.ts. (The example test code is in ./server/example/example.test.ts)

You might use the following code snippet:
```typescript
child_process.spwanSync('./absolute', ['build_server']);   // This will create out/ directory
fs.existsSync('./out');   // You can check whether the out directory exists
child_process.spawnSync('./absolute', ['clean']);    // This will remove out/ directory
fs.existsSync('./out');   // You can check whether the out directory exists
```

Reply:
Okay. I modified argument value to remove the out directory (+ files and sub-directories).
I also added a test code file named clean.test.ts to bootstrap/test/.
Thank you for review :D